### PR TITLE
Add pipreqs recipe

### PR DESCRIPTION
**Pull request by proinsias**

### CODE REVIEW

File: recipes/pipreqs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pipreqs=pipreqs.pipreqs:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - docopt
+    - yarg
```
Comment (jakirkham) [inline recipes/pipreqs/meta.yaml]: Please drop run time dependencies from build if possible (excepting `python` and potentially `setuptools`).

File: recipes/pipreqs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+test:
+  imports:
+    - pipreqs
+
+  commands:
+    - pipreqs --help
+
+about:
+  home: https://github.com/bndr/pipreqs
+  license: Apache
```
Comment (jakirkham) [inline recipes/pipreqs/meta.yaml]: Please add the version.

**Comment by proinsias** [comment]
@jakirkham – Updated license to Apache-2.0.

**Comment by jakirkham** [comment]
Thanks @proinsias.

**Pull request merged**